mlflow/sklearn/__init__.py

```diff
@@ -1333,14 +1333,23 @@ def get_input_example():
                 flavor_name, "registered_model_name", None
             )
             model_format = get_autologging_config(flavor_name, "model_format", "xgb")
```

Comment:
Nit: can we pull this into the `if` block?

Reply:
good idea :)